src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx

```diff
@@ -96,11 +93,16 @@ const EquipmentFilter = forwardRef(
         // fetching options in different criterias
         useEffect(() => {
             // Load voltage level IDs
-            fetchVoltageLevelsListInfos(studyUuid, currentNode.id)
+            fetchVoltageLevelsMapInfos(studyUuid, currentNode.id)
                 .then((voltageLevels) => {
                     const vlMap = new Map();
-                    voltageLevels.forEach((vl) => vlMap.set(vl.id, vl.name));
+                    const nvSet = new Set();
+                    voltageLevels.forEach((vl) => {
+                        vlMap.set(vl.id, vl.name);
```

Comment:
Why 2 collections ?
Just one map : id -> vl ?

Reply:
must aggregate nominal V to show in the separated criteria..

Reply:
The Set contains only unique elements